bot/src/bot.rs

```diff
@@ -220,10 +220,12 @@ fn autocompound_instance(daemon: &Daemon, instance: (&str, &Addr)) -> anyhow::Re
     app.set_address(address);
     use carrot_app::AppQueryMsgFns;
     let resp: CompoundStatusResponse = app.compound_status()?;
+    let pool_rewards: AvailableRewardsResponse = app.available_rewards()?;
```

Comment:
We should combine both queries into one so we only have to do one RPC request.

Reply:
> We should combine both queries into one so we only have to do one RPC request.

Sure

Reply:
Added that!